### ISSUE TITLE
SSE error message appears to be wrong ("err" vs. "error")

### ISSUE THREAD

**Issue by graydon** [stellar/go#883]
Reported on chat by @cballou

So services/horizon/internal/render/sse/main.go handles errors as follows:

~~~
// WriteEvent does the actual work of formatting an SSE compliant message
// sending it over the provided ResponseWriter and flushing.
func WriteEvent(ctx context.Context, w http.ResponseWriter, e Event) {
        if e.Error != nil {
                fmt.Fprint(w, "event: err\n")
~~~

The W3C spec for handling errors is to use `event: error:`
https://www.w3.org/TR/2009/WD-eventsource-20090421/
I would surmise that this is a bug unless there's a different spec to follow
In other words, client EventSource handlers have no knowledge of err events.
Because they're built to spec.
It's also obvious it's unique to Stellar Core if you just google `"event: err\n"` vs `"event: error\n"`

**Comment by cballou** [comment]
Accompanying pull request:
https://github.com/stellar/go/pull/884

**Comment by bartekn** [comment]
Merged #884.
